test/suites/recover.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+test_recover() {
+  reset_systems 4 0 0
+
+  systems=("micro01" "micro02" "micro03" "micro04")
+
+  lxc exec micro01 -- sh -c "TEST_CONSOLE=0 microcloud init --auto > out"
+  for m in "${systems[@]}" ; do
+    validate_system_lxd "${m}" 4
+    validate_system_microceph "${m}"
+    validate_system_microovn "${m}"
+  done
+
+  # MicroCluster takes a while to update the core_cluster_members table
+  while lxc exec micro01 -- microcloud cluster list -f csv | grep -q PENDING; do
+    sleep 2
+  done
+
+  for m in "${systems[@]}"; do
+    lxc exec "${m}" -- sudo snap stop microcloud
+  done
+
+  lxc exec micro01 -- snap install yq
```

Comment:
I realize that this is probably only here for the sake of efficiency but it's likely to get lost as the test suite grows, so we should probably add it to the setup step [here](https://github.com/canonical/microcloud/blob/359cd469a36f52a950f24ec9945f18f22e74d039/test/includes/microcloud.sh#L982-L989)